charts/camunda-platform-8.9/test/unit/orchestration/configmap_unified_test.go

```diff
@@ -568,3 +579,81 @@ func (s *ConfigmapTemplateTest) TestDifferentValuesInputsUnifiedRDBMS() {
 
 	testhelpers.RunTestCases(s.T(), s.chartPath, s.release, s.namespace, s.templates, testCases)
 }
+
+func (s *ConfigmapTemplateTest) TestApplicationYamlMultiregion() {
+	testCases := []testhelpers.TestCase{
+		{
+			Name: "TestApplicationYamlShouldContainMultiregion",
+			Values: map[string]string{
+				"global.multiregion.regions":  "2",
+				"global.multiregion.regionId": "0",
+			},
+			RenderTemplateExtraArgs: []string{
+				"--set-string", "orchestration.clusterSize=8",
+				"--set-string", "orchestration.partitionCount=8",
+				"--set-string", "orchestration.replicationFactor=4",
+			},
+			Verifier: func(t *testing.T, output string, err error) {
+				require.NoError(t, err, "Template rendering should not fail")
+
+				var configmap corev1.ConfigMap
+				helm.UnmarshalK8SYaml(t, output, &configmap)
+
+				var configmapApplication OrchestrationConfigYAML
+				err = yaml.Unmarshal([]byte(configmap.Data["application.yaml"]), &configmapApplication)
+				require.NoError(t, err, "Failed to unmarshal application.yaml")
+
+				// Verify that we have 4 initial-contact-points (clusterSize=8, regions=2, so 8/2=4 local pods)
+				initialContactPoints := configmapApplication.Camunda.Cluster.InitialContactPoints
+				require.Len(t, initialContactPoints, 4, "Should have 4 initial-contact-points for clusterSize=8 with 2 regions")
+
+				// Verify the actual values match expected format
+				expectedPrefix := s.release + "-zeebe-"
+				expectedSuffix := ".${K8S_SERVICE_NAME}:26502"
+				expectedContactPoints := []string{
+					expectedPrefix + "0" + expectedSuffix,
+					expectedPrefix + "1" + expectedSuffix,
+					expectedPrefix + "2" + expectedSuffix,
+					expectedPrefix + "3" + expectedSuffix,
```

Comment:
If the number of regions in this test is 2 , does that mean that the initialContactPoints should only have the odd numbers of zeebe nodes, rather than all 4?

otherwise why have the  division in the implementation?